.github/workflows/firefly-ci.yml

```diff
@@ -20,8 +20,5 @@ jobs:
       - name: Install Dependencies
         run: yarn --ignore-scripts
 
-      - name: Check Formatting
```

Comment:
ESLint and Prettier have slightly conflicting configurations, so the formatting check will _always_ fail (since we use Husky and pre-commit hooks for linting).

Reply:
Hmm, that sounds like something we should resolve though. Maybe we should try something like https://github.com/prettier/prettier-eslint or figure out how to match the Prettier and ESLint configurations?